src/ocm/services/ClustersService.ts

```diff
@@ -22,6 +23,27 @@ const ClustersService = {
     const fileName = contentHeader?.match(/filename="(.+)"/)?.[1];
     return { data, fileName };
   },
+
+  async update(cluster: Cluster, params: V2ClusterUpdateParams) {
+    if (ocmClient) {
+      params = ClustersService.updateClusterTags(cluster.tags, params);
+    }
+    return ClustersAPI.update(cluster.id, params);
+  },
+
+  updateClusterTags(
+    clusterTags: Cluster['tags'],
+    params: V2ClusterUpdateParams,
+  ): V2ClusterUpdateParams {
+    const tags = clusterTags?.split(',') || [];
+    if (tags?.includes(AI_UI_TAG)) {
+      params.tags = clusterTags;
+    } else {
+      tags?.push(AI_UI_TAG);
+      params.tags = tags?.join(',');
```

Comment:
Maybe we could only PATCH sending the `tags` property if the `AI_UI_TAG` is absent.
That would reduce the probability of clashes.

Reply:
Good idea :+1:

Reply:
@jgyselov I can't see the change on this, is it pushed?

Reply:
No, I didn't push the changes yet. I'll do that once I add the tags on install.